custom_components/flexmeasures_hacs/config_flow.py

```diff
@@ -1,58 +1,88 @@
 """Config flow for FlexMeasures integration."""
+
 from __future__ import annotations
 
-import logging
-from typing import Any
+from collections.abc import Mapping
+from typing import Any, cast
 
 from flexmeasures_client import FlexMeasuresClient
+from flexmeasures_client.exceptions import EmailValidationError
 import voluptuous as vol
 
-from homeassistant import config_entries
-from homeassistant.config_entries import ConfigEntry, OptionsFlow
-from homeassistant.core import HomeAssistant, callback
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import async_get_hass
 from homeassistant.data_entry_flow import FlowResult
-from homeassistant.exceptions import HomeAssistantError
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaCommonFlowHandler,
+    SchemaConfigFlowHandler,
+    SchemaFlowError,
+    SchemaFlowFormStep,
+)
 
 from .const import DOMAIN
 
-_LOGGER = logging.getLogger(__name__)
-
-STEP_USER_DATA_SCHEMA = vol.Schema(
+SCHEMA = vol.Schema(
     {
+        vol.Required("url", default="https://seita.energy"): str,
         vol.Required(
-            "url", description={"suggested_value": "http://localhost:5000"}
+            "username",
+            default="example@example.com",
         ): str,
-        vol.Required(
-            "username", description={"suggested_value": "toy-user@flexmeasures.io"}
-        ): str,
-        vol.Required("password"): str,
-        vol.Required("power_sensor", description={"suggested_value": 1}): int,
-        vol.Required(
+        vol.Required("password", default="password"): str,
+        vol.Optional("power_sensor", default=5): int,
+        vol.Optional("soc_minima_sensor_id", default=5): int,
+        vol.Optional("soc_maxima_sensor_id", default=5): int,
+        vol.Optional("fill_level_sensor_id", default=5): int,
+        vol.Optional("fill_rate_sensor_id", default=5): int,
+        vol.Optional("usage_forecast_sensor_id", default=5): int,
+        vol.Optional("thp_fill_rate_sensor_id", default=5): int,
+        vol.Optional("thp_efficiency_sensor_id", default=5): int,
+        vol.Optional("nes_fill_rate_sensor_id", default=5): int,
+        vol.Optional("nes_efficiency_sensor_id", default=5): int,
+        vol.Optional("rm_discharge_sensor_id", default=5): int,
+        vol.Optional("schedule_duration", default="PT24H"): str,
+        vol.Optional(
             "consumption_price_sensor", description={"suggested_value": 2}
```

Comment:
For testing purposes in the short term, it may be convenient to have some cleverly chosen IDs here.